yb-voyager/cmd/exportSchema.go

```diff
@@ -72,12 +73,19 @@ func exportSchema() {
 		utils.ErrExit("Failed to connect to the source db: %s", err)
 	}
 	source.DB().CheckRequiredToolsAreInstalled()
+	sourceDBVersion := source.DB().GetVersion()
 
-	fmt.Printf("%s version: %s\n", source.DBType, source.DB().GetVersion())
+	fmt.Printf("%s version: %s\n", source.DBType, sourceDBVersion)
 
 	CreateMigrationProjectIfNotExists(source.DBType, exportDir)
 	source.DB().ExportSchema(exportDir)
 	utils.PrintAndLog("\nExported schema files created under directory: %s\n", exportDir+"/schema")
+
+	callhome.InitJSON(exportDir)
+	payload := callhome.GetPayload()
+	payload.SourceDBType = source.DBType
+	payload.SourceDBVersion = sourceDBVersion
+	callhome.PackAndSendPayload(exportDir)
```

Comment:
@rahulb-yb what is the expected time for this function call to complete - from same network and different(public) network?
Can we consider making it a goroutine to avoid delays to user experience?

Reply:
Or are we waiting for the response to come, can we do something some just send the request and don't wait for the response? 
referred: [StackOverflow link](https://stackoverflow.com/questions/2190854/sending-post-requests-without-waiting-for-response#:~:text=You%20cannot%20just%20send%20data,will%20wait%20for%20that%20response.)

Reply:
I do not have an estimate over the algorithm/time complexity used, in my current tests it seems to not affect performance significantly- Even if I did use a goroutine, we would have to ask the main function to wait on the goroutine to finish to as to safely update the diagnostics.json file, and actually send the data

I will also make sure once again that I have only packed/sent the diagnostics data after all the crucial steps have been completed corresponding to each phase of usage of yb-voyager

Reply:
yeah in the first option you will still have to wait for the goroutine to complete. Check if there is a easy way to not wait for the response of POST request made.

Reply:
As far as my knowledge of computer networking goes, HTTP requests are always met with a response...
Any other solution would involve not using the standard libraries available afaik

Reply:
ok, just having a idea about how much time it is taking will be good. Also check if we have or can have some timeout for HTTP requests so that its not waiting too long for this.

Reply:
I do not see any reason why we would face an HTTP timeout, unless someone stops the already running services- just did some testing and sending the request (and receiving the response) seems to take not more than ~150ms. This was done with a fully populated payload, and also includes the time taken to "save" the json file locally